### ISSUE TITLE
CVE summary for java deserialiazation

### ISSUE THREAD

**Issue by mudongliang** [GrrrDog/Java-Deserialization-Cheat-Sheet#4]
I have read through all the README. There are two parts where contain CVEs for java deserialization - `Exploits` and `Vulnerable apps`.
Why do you separate them info two parts? I think we can combine them into one part which is based on CVE, or vulnerable app name. It may be a long list with more 30 items, but it is more clear.
Additionally, how do you collect all those vulnerabilities in the README? It is a heavy but meaningful work. I think it's best if we can collect all the CVEs about java deserialization.

**Comment by GrrrDog** [comment]
1) the primary goal of this cheat sheet is to help pentesters. So Exploits is a list of vulnerable apps with enough details for exploitation or with public sploits. Vulnerable apps is just a list of apps with javadeser vulns, but without all necessary info for exploitation.
2) A part of this list I've got from @frohoff Another part I've got from monitoring Twitter and gitter. From time to time I use search at mitre.org. 
I'd like to collect all the CVEs, but there are some problems: not all vendors get CVEs for vulns in thier products; some vendors give only a list of vulnerable apps.

I like javadeser topic. It helps me much during pentests. So it's not hard for me :) However, I'd like to work together. Mb we could discuss some details in faster way in Twitter?
